app/models/alchemy/picture_version.rb

```diff
@@ -29,4 +35,8 @@ def store_and_save!(image)
     self.save
     image
   end
+
+  def remove_from_store
+    Dragonfly.app(:alchemy_pictures).datastore.destroy(file_uuid)
```

Comment:
This is not done automatically when destroying a version!?

Reply:
nope. The 'PictureVersion' only interacts with the Dragonfly datastore when storing images, then records the resulting 'file_uuid' in the DB. Destroying then only removes the record from the DB.